invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -113,6 +114,49 @@ export class RDMDepositForm extends Component {
             { text: "Other", value: "other" },
           ],
         },
+
+        // TODO: Replace with an API backend
```

Comment:
minor: even if I agree with the TODO, in my experience they tend to fall in the forgotten limbo... Openining issues seems more effective.

Reply:
You're right, we keep forgetting them. I made https://github.com/inveniosoftware/react-invenio-deposit/issues/82 for the more general case.